src/meta/src/hummock/manager/mod.rs

```diff
@@ -1056,11 +1056,10 @@ impl HummockManager {
         } else {
             table_to_vnode_partition
                 .retain(|table_id, _| compact_task.existing_table_ids.contains(table_id));
-            if group_config.compaction_config.split_weight_by_vnode > 0 {
+            if compact_task.existing_table_ids.len() == 1 {
```

Comment:
It will break the partition of hybrid cg like cg2, cg3. why should we limit the existing_table_ids size ?

Reply:
Same question here.

Reply:
Agree, I will fix it